### DISCUSSION TITLE
Atom Lite Reboot Loop

### DISCUSSION THREAD

**Discussion by thezfunk** [esphome-econet/esphome-econet#386] [General]
I installed the package off the webpage to my Atom Lite and the logs show it is just in a reboot loop and I have no idea why.

`ELF file SHA256: 0000000000000000

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 188777542, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4
[E][ota:490]: Boot loop detected. Proceeding to safe mode.
[E][ota:159]: Error set: unspecified
[W][mdns:019]: mDNS init failed: ESP_ERR_INVALID_STATE
[E][component:114]: Component mdns was marked as failed.
[E][mdns:159]: Error set: unspecified

assert failed: tcpip_send_msg_wait_sem IDF/components/lwip/lwip/src/api/tcpip.c:455 (Invalid mbox)

Backtrace:0x400837a5:0x3ffb23700x4008b6b5:0x3ffb2390 0x40090bed:0x3ffb23b0 0x40104d0d:0x3ffb24e0 0x4011520d:0x3ffb2510 0x4011526d:0x3ffb2530 0x401043b9:0x3ffb2580 0x400e24ed:0x3ffb25a0 0x400e2526:0x3ffb25c0 0x400e11c5:0x3ffb25e0 0x401682b9:0x3ffb2620 0x40168355:0x3ffb2640 0x400e6170:0x3ffb2660 0x400e1500:0x3ffb2690 0x400e9499:0x3ffb26d0 0x400efc7e:0x3ffb2820 

ELF file SHA256: 0000000000000000

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 188777542, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13132
load:0x40080400,len:3036
entry 0x400805e4
[W][mdns:019]: mDNS init failed: ESP_ERR_INVALID_STATE
[E][component:114]: Component mdns was marked as failed.
[E][mdns:159]: Error set: unspecified

assert failed: tcpip_send_msg_wait_sem IDF/components/lwip/lwip/src/api/tcpip.c:455 (Invalid mbox)

Backtrace:0x400837a5:0x3ffb23b00x4008b6b5:0x3ffb23d0 0x40090bed:0x3ffb23f0 0x40104d0d:0x3ffb2520 0x4011520d:0x3ffb2550 0x4011526d:0x3ffb2570 0x401043b9:0x3ffb25c0 0x400e24ed:0x3ffb25e0 0x400e2526:0x3ffb2600 0x400e11c5:0x3ffb2620 0x401682b9:0x3ffb2660 0x40168355:0x3ffb2680 0x400e6170:0x3ffb26a0 0x400eb565:0x3ffb26d0 0x400efc7e:0x3ffb2820 
`

**Comment by tronikos** [comment]
See https://github.com/esphome-econet/esphome-econet/issues/379
